spotbugs-tests/src/test/java/edu/umd/cs/findbugs/ba/Issue408Test.java

```diff
@@ -0,0 +1,26 @@
+package edu.umd.cs.findbugs.ba;
+
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assume.assumeThat;
+
+import org.junit.Test;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+
+/**
+ * @see <a href="https://github.com/spotbugs/spotbugs/issues/408">#408</a>
+ * @since 3.1
+ */
+public class Issue408Test extends AbstractIntegrationTest {
+
+    @Test
+    public void test() {
+        assumeThat(System.getProperty("java.specification.version"), is("9"));
+
+        performAnalysis("../java9/module-info.class");
```

Comment:
This will not work, because the module-info class file is ignored by SpotBugs now. Either we must specify at least one class more or change the test runner to expect errors.